Assets/_GameJamCat/Scripts/UI/DossierMenuView.cs

```diff
@@ -23,16 +26,28 @@ public class DossierMenuView : MonoBehaviour
         [SerializeField] private Image _catActivitiesImage = null;
         [SerializeField] private TMP_Text _cativities = null;
 
+        [Title("Sprites")]
+        [SerializeField] Sprite _tunaIcon;
+        [SerializeField] Sprite _salmonIcon, _chickenIcon, _pumpkinIcon, _dryKibbleIcon, _wetKibbleIcon, _kibbleIcon, _boneIcon, _beefIcon = null;
+        [SerializeField] Sprite _yarnBallsIcon, _cardboardBoxIcon, _fishingRodIcon, _catnipSackIcon, _hijinksIcon, _scratchingPostIcon, _laserIcon, _tennisBall = null;
```

Comment:
Do you think it would be better to serialize a dictionary of <string, Sprite>? Or do we know if this is the complete list and dont really have to worry about the list expanding?

Reply:
I was actually originally thinking of a serialized stuct, same with toys/activities


public Food {
   enum,
  string, 
Sprite
}

But it's up to you guys

Reply:
Per the spreadsheet, this is the complete list and we don't need to worry about it expanding. Are we using the icons any where else? If they go somewhere other than the dossier I'd get having a struct but otherwise I'm not sure I see the point